pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/MultiStageOperator.java

```diff
@@ -45,6 +46,10 @@ public abstract class MultiStageOperator implements Operator<TransferableBlock>,
   protected final Map<String, OperatorStats> _operatorStatsMap;
   private final String _operatorId;
 
+  public MultiStageOperator(OperatorExecutionContext context) {
+    this(context.getRequestId(), context.getStageId(), context.getServer());
+  }
```

Comment:
directly take the context object is fine. no need to parse the 3 components out.